src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -46,21 +47,16 @@ var (
 	m                   *yamlconf.Manager
 	lastCloudConfigETag = map[string]string{}
 	r                   = regexp.MustCompile("\\d+\\.\\d+")
-	// Request the config via either chained servers or direct fronted servers.
-	cf = util.NewChainedAndFronted()
```

Comment:
Oh here too re-creating chained and fronted each time will lose its internal state that will only use direct chained proxies if they're working.